2_Operations/user-stories.md

```diff
@@ -2,9 +2,17 @@
 
 User Stories are well-defined tasks to be done by a Nitean that have a specific, measurable goal and impact. The vast majority of issues are written as User Stories, and only issues created by support that are reactionary and straightforward are not. Support issues are also not included in sprints.
 
+All user stories are located in the project repository:
+- EBN project: `niteoweb/easyblognetworks`
+- KAI project: `niteoweb/kai`
+- WooCart project: `niteoweb/woocart`
+- "Niteo project" (company stuff): `niteoweb/operations`
+
+All projects have multiple repositories (e.g. EBN https://github.com/niteoweb/easyblognetworks/#repositories) but only one project repository. The idea behind this is to have all user stories for one project in one repository.
```

Comment:
Re-read this, it doesn't make much sense.

Reply:
We have many repositories for one project (e.g. ebn) but only one repository is the repository where we keep project user stories.

Reply:
> We have many repositories for each project but we keep User Stories in only one, main, repository.